src/main/java/racingcar/domain/RacingCarsGroup.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCarsGroup {
+
+    private List<Car> cars = new ArrayList<>();
```

Comment:
이 필드는 추후 변동 가능성이 있어서 final 지정자를 설정안한건가요?!

Reply:
final로 선언하면 모든 생성자에서 초기화를 해야 한다는 것을 몰랐었고, "오류가 뜨네? 그럼 final을 지우자"라고 안일한 생각을 했었네요. final 붙이고
```suggestion
    public RacingCarsGroup() {
        this.cars = new ArrayList<>();
    }
```
이런 식으로 해결 가능함을 확인했습니다!
지적해주셔서 감사드립니다.